rbc/omnisci_backend/omnisci_buffer.py

```diff
@@ -347,19 +349,94 @@ def omnisci_buffer_setitem(a, i, v):
 
 
 @extending.intrinsic
-def omnisci_buffer_is_null_(typingctx, data):
-    sig = types.int8(data)
+def omnisci_column_set_null_(typingctx, col_var, row_idx):
```

Comment:
This is Column specific implementation and should be implemented in `omnisci_column.py`